python/pyspark/tests.py

```diff
@@ -1515,12 +1515,12 @@ def test_oldhadoop(self):
 
         conf = {
             "mapred.output.format.class": "org.apache.hadoop.mapred.SequenceFileOutputFormat",
```

Comment:
@srowen Thanks, you are right. see:
https://github.com/apache/hadoop/blob/release-3.0.0-alpha2-RC0/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Job.java#L1296

if set `mapred.reducer.new-api=true`, the exception:
```
Exception in thread "main" java.io.IOException: mapred.output.format.class is incompatible with new reduce API mode.
	at org.apache.hadoop.mapreduce.Job.ensureNotSet(Job.java:1210)
	at org.apache.hadoop.mapreduce.Job.setUseNewAPI(Job.java:1258)
	at org.apache.hadoop.mapreduce.Job.submit(Job.java:1299)
	at org.apache.hadoop.mapreduce.Job.waitForCompletion(Job.java:1324)
	at wpmcn.hadoop.WordCount.main(WordCount.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:221)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
```
I will update it soon.